### ISSUE TITLE
Limit on Confirmed orders should not be 500

### ISSUE THREAD

**Issue by crayonflakesmanu** [StompMarket/helpdesk#924]
![screenshot 2019-02-05 at 1 28 16 pm](https://user-images.githubusercontent.com/39734400/52259825-0170d000-294a-11e9-9fdd-e06db49ca79c.png)

**Comment by anban4u** [comment]
Agreed. Lets ramp it up to 2000. Also, we probably need a date/order filter on this screen.
